src/components/Stories/Stories.tsx

```diff
@@ -29,10 +33,15 @@ export function Stories({
     items,
     onPreviousClick,
     onNextClick,
-    initialStoryIndex = 0,
+    initialStoryIndex,
+    index = 0,
     disableOutsideClick = true,
 }: StoriesProps) {
-    const [storyIndex, setStoryIndex] = React.useState(initialStoryIndex);
+    const [storyIndex, setStoryIndex] = React.useState(initialStoryIndex ?? index);
+
+    React.useEffect(() => {
+        setStoryIndex(index);
```

Comment:
This code will overwrite storyIndex that is set via the property `initialStoryIndex`

Reply:
Yep. `index` is new field and it stronger than deprecated `initialStoryIndex`

Reply:
[30e356c](https://github.com/gravity-ui/uikit/pull/464/commits/30e356cbd4c62e136e7db490371e1b51dc9b3c86)